Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.List.ts2

```diff
@@ -45,4 +50,14 @@ prototype(Neos.NeosIo:Reference.List.Item) < prototype(TYPO3.TypoScript:Template
 		}
 	}
 	projectType = ${q(q(node).property('projectType')).property('title')}
+
+	@cache {
```

Comment:
@kitsunet My question is more about line 35ff which turns the whole list `uncached`. can the `dynamic` mode be used to cache by default and only use uncached if the `search` query argument is set?